gateway-ha/src/main/java/io/trino/gateway/ha/config/MonitorConfiguration.java

```diff
@@ -37,6 +38,8 @@ public class MonitorConfiguration
 
     private String queuedQueriesMetricName = "trino_execution_name_QueryManager_QueuedQueries";
 
+    private Duration backendMetricsRegistryRefreshPeriod = DEFAULT_BACKEND_METRICS_REGISTRY_REFRESH_PERIOD;
+
```

Comment:
Any reason to have the default as a constant over in `BackendsMetricStats` instead of keeping it here, like we do for the other configs (e.g. `queryTimeout`)?